### ISSUE TITLE
Win11: NVDA is not announcing the available suggestions for “Find a setting” search field in settings page  

### ISSUE THREAD

**Issue by SMTestGH** [nvaccess/nvda#12758]
### Steps to reproduce:
1. Open Settings with Win+I
2. Tab to search field, type "test"
3. Search results are displayed

### Actual behavior:
NVDA does not read that suggestions are available and does not read first item

### Expected behavior:
NVDA should announce that suggestions are available and read the first suggestion
Note: both JAWS and Narrator work as expected.

#### NVDA installed/portable/running from source:
Installed/portable both repro the same issue

#### NVDA version:
2021.1

#### Windows version:
Windows 11 Insider builds

#### Name and version of other software in use when reproducing the issue:
N/A

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No. Both Narrator and JAWS work as expected.
#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by josephsl** [comment]
Hi, Windows App Essentials add-on should resolve this. Thanks.

**Comment by josephsl** [comment]
Hi,

Note that Settings app suggestions list is similar to Start suggestions list but it is not - it requires NVDA to listen to UIA layout invalidated event for proper functionality. I am willing to donate this part of Windows App Essentials add-on to NV Access provided that we come up with a baseline user experience notes.

Thanks.

**Comment by josephsl** [comment]
Hi,

An important point to consider: Narrator does not announce the top suggestion - it will announce suggestion count followed by "suggestion". My suggestion about using Windows App Essentials add-on to test this still stands.

Thanks.

**Comment by josephsl** [comment]
Hi,

Another important point to consider: this issue is not limited to Windows 11 - it also happens with Windows 10 and it happens for suggestions list that raises UIA layout invalidated event.

I think there is an issue concerning this problem (if found, this issue will be tagged and might be closed as a duplicate).

Thanks.

**Comment by josephsl** [comment]
Hi,

And found it: #7330 - I may need to raise a summary issue for these two at some point - implementation is ready to go, as what's needed to be done is separate the responsible component from Windows App Essentials add-on and transplant it inside UIA objects mix-in.

Thanks.
